build.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash -xe
+export CMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE:-Release}
```

Comment:
The only issue I have with this, we now have defaults for those values specified twice. I don't see any solution for that.

Reply:
I moved them to build_script.sh. Other places are probably not needed.